src/screens/Login/Login.style.ts

```diff
@@ -0,0 +1,63 @@
+import { createStyleSheet } from 'react-native-unistyles';
+
+export const styleSheet = createStyleSheet((theme) => ({
+	container: {
+		position: 'relative',
```

Comment:
If I remember correctly, position is relative by default. There is hardly any point in this style prop